### PR TITLE
pass revision-1

### PR DESCRIPTION
**Pull request by kenji21**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

TL; DR: make pass get password quicker to have ansible job run faster

Before this revision:

```
$ time pass show site.com/login >/dev/null

real	0m1.509s
user	0m0.500s
sys	0m0.411s
$ time pass show site.com/login >/dev/null

real	0m1.339s
user	0m0.526s
sys	0m0.428s
```

Run `brew reinstall --build-from-source pass`, and then:

```
$ time pass show site.com/login >/dev/null
real	0m0.156s
user	0m0.018s
sys	0m0.016s
$ time pass show site.com/login >/dev/null

real	0m0.110s
user	0m0.020s
sys	0m0.020s
```

Most of time is consumed by `brew --prefix gnu-getopt` due to this issue: https://github.com/Homebrew/brew/issues/8189

### CODE REVIEW

**Comment by fxcoudert** [comment]
We really don't want to go replace things in various shell scripts. This should be reported as a bug in pass, and fixed there: they can record the getopt location at build time, or use `brew --prefix`.

**Comment by kenji21** [comment]
sure, but a change already has been asked there, and no new release came out since: https://lists.zx2c4.com/pipermail/password-store/2018-October/003421.html

**Comment by fxcoudert** [comment]
I understand, but this is exactly why we shouldn't take it: we don't want to become de facto maintainers of every formula we ship. We will consider shipping patches if they are accepted by upstream and the formula is otherwise severely broken or unbuildable. Your best bet is to ask upstream to integrate the change and make a new release.

**Comment by kenji21** [comment]
Ok, I understand, found that a fix is already in the pass repo (https://git.zx2c4.com/password-store/commit/?id=07b169ec32ad6961ed8625a0b932a663abcb01d2)

To get pass faster, we can simply install current HEAD:
```
brew remove pass ; brew install pass --HEAD
```

**Comment by kenji21** [comment]
Maybe PR like this one can be prevented by adding a line about the `brew install --HEAD foo` options in [CONTRIBUTING.md](https://github.com/homebrew/homebrew-core/blob/master/CONTRIBUTING.md) ?

**Comment by SMillerDev** [comment]
We don't support head builds, that's in our overall documentation. Using the upstream commit as a patch would be an acceptable solution though, even though asking for a release would be better.
